howto/instance/access-instance.md

```diff
@@ -1,9 +1,66 @@
 (howto-access-instance)=
 # How to access an instance
 
-In some cases, it might be necessary to access an individual instance for debugging reasons.
+In some cases, it might be necessary to access an individual instance for debugging or development purposes.
 
-You can do this on the command line with the `amc` command, or you can use [scrcpy](https://github.com/Genymobile/scrcpy) for graphical access.
+You can:
+- Run `anbox-connect` for secure ADB connections.
+- Access an instance using the `amc` command for shell access.
+- Use [scrcpy](https://github.com/Genymobile/scrcpy) for graphical access to Android instance.
+
+## Access Android instance using anbox-connect
+
+Secure ADB connection is a feature introduced in Anbox Cloud 1.23.0 that allows users to create a secure ADB connection through the `anbox-connect` command without exposing the [ADB service](https://anbox-cloud.io/docs/howto/instance/expose-services).
+
+### Install anbox-connect snap
+
+First, install the `anbox-connect` snap from the snap store:
+
+    snap install anbox-connect
+
+### Generate a presigned connection URL
+
+Generate a presigned connection URL in the following ways:
+
+- **From the Dashboard**:
+  Navigate to the `Instances` view from the left panel, locate a running instance, click the ![Connect ADB|16x16](https://assets.ubuntu.com/v1/tbd_connect-adb.png) icon button, then click `Authorize` in the popup dialog and copy the command provided.
```

Comment:
Guess the icon needs to be uploaded to assets manager. Adding this comment so that we keep track of that.

Reply:
Yes, I don't have the api token and needs your help to upload the image to the asset manager.

Reply:
Use this link: https://assets.ubuntu.com/v1/51973cea-adb-connect-icon.png

Reply:
Thanks for the uploading